readthedocs/core/static-src/core/js/doc-embed/search.js

```diff
@@ -61,7 +72,7 @@ function attach_elastic_search_query_sphinx(data) {
                     for (var i = 0; i < results.length; i += 1) {
                         var result = results[i];
                         var blocks = result.blocks;
-                        var list_item = $('<li style="display: none;"></li>');
```

Comment:
`slideDown` was removed, so this element can't be hidden (it could, but I'm not really sure if we need it? I didn't notice any difference between with/out slidedown)